components/roll/actions/create-company/create-company.mjs

```diff
@@ -0,0 +1,112 @@
+import _ from "lodash";
+import roll from "../../roll.app.mjs";
+
+export default {
+  key: "roll-create-company",
+  name: "Create Company",
+  version: "0.0.1",
+  description: "Create a new company [See the docs here](https://docs.rollhq.com/docs/roll-api#api-url)",
+  type: "action",
+  props: {
+    roll,
+    name: {
+      type: "string",
+      label: "Name",
+      description: "The company's name.",
+      optional: true,
+    },
+    status: {
+      type: "string",
+      label: "Status",
+      description: "The company's status.",
+      optional: true,
+    },
```

Comment:
It looks like several of the action props are used in multiple actions. If a prop (for example, `name`) is used in multiple components, it can be stored in the app file (roll.app.mjs) and referred to by its propDefinition.
```suggestion
    name: {
      propDefiniton: [
         "roll",
         name,
      ],
    },
    status: {
      propDefinition: [
         roll,
         "status"
      ],
    },
```